src/index.html

```diff
@@ -7,8 +7,49 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img src="./images/logo.png" alt="logo  MOYO">
+      </a>
+      <nav class="nav-bar">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="#" class="nav-link is-active">Apple</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Samsung</a>
+          </li>
```

Comment:
Please, add spaces between siblings, check it for all your html elements
```suggestion
          <li class="nav-item">
            <a href="#" class="nav-link is-active">Apple</a>
          </li>
          
          <li class="nav-item">
            <a href="#" class="nav-link">Samsung</a>
          </li>
```